src/encoded/schemas/target.json

```diff
@@ -99,12 +107,13 @@
         }
     },
     "boost_values": {
-        "gene_name": 1.0, 
-        "label": 1.0, 
-        "dbxref": 1.0, 
-        "aliases": 1.0, 
-        "organism.name": 1.0,  
-        "organism.scientific_name": 1.0, 
+        "gene.symbol": 1.0,
```

Comment:
add "gene.symbol" to facets - this would let us group H3 modifications together.

Reply:
Added in ENCD-3998-modification-1 commit.